docs/pages/application-access/getting-started.mdx

```diff
@@ -77,19 +77,32 @@ Install Teleport on the host where you will run the Teleport Application Service
 
 Now start Teleport and point it to the application endpoint:
 
-```code
-$ sudo teleport app start \
-  --name=grafana \
-  --token=/tmp/token \
-  --uri=http://localhost:3000 \
-  --auth-server=https://teleport.example.com:3080
+This can be done by adding the application service to your teleport config, typically located at `/etc/teleport.yaml`.
+
+Add these lines to your config:
```

Comment:
Let's show a complete config with auth token and proxy server set (to make this equivalent to the previous `teleport app start` command)